definitions/ext-dell_avamar/definition.yml

```diff
@@ -0,0 +1,19 @@
+domain: EXT
+type: DELL_AVAMAR
+
+synthesis:
+  rules:
+  - identifier: clusterCI
```

Comment:
Just double checking: Would this identifier be unique on a per-account basis ?

Reply:
Correct